### ISSUE TITLE
[BUG] false positive on H011

### ISSUE THREAD

**Issue by christopherpickering** [Riverside-Healthcare/djLint#112]
```html
<a href="{{ url_for('connection_bp.one_connection', connection_id=connection.id) }}">{{ connection }}</a>
```
Triggers a false positive of H011.
